Gemfile.lock

```diff
@@ -2,71 +2,131 @@ GEM
   remote: http://rubygems.org/
```

Comment:
I have no idea about the GH stack but is it really required to commit Gemfile.lock into this repo?